Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -147,6 +148,27 @@ class TIFFLoadingContext {
             return Color(first_component, second_component, third_component, alpha);
         }
 
+        if (m_metadata.photometric_interpretation() == PhotometricInterpretation::RGBPalette) {
+            auto const index = TRY(stream.read_bits<u16>(bits_per_sample[0]));
+            auto const alpha = TRY(manage_extra_channels());
+
+            // SamplesPerPixel == 1 is a requirement for RGBPalette
+            // From description of PhotometricInterpretation in Section 8: Baseline Field Reference Guide
```

Comment:
Can you also add the direct quote:

```
// "In a TIFF ColorMap, all the Red values come first, followed by the Green values,
//  then the Blue values."
```

please? (Because it's surprising that the channels are separated like that – no other format stores its palette that way as far as I know.)